src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+const resolveMessage = `
+  <div class="message">
+    Promise was resolved!
+  </div>
+`;
+const rejectMessage = `
+  <div class="message error-message">
+    Promise was rejected!
+  </div>
+`;
+const body = document.getElementById('body');
+
+const createPromise = () => {
+  const logo = document.querySelector('.logo');
+
+  const resolver = (resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve(resolveMessage);
+    });
+
+    setTimeout(() => {
+      reject(rejectMessage);
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+};
```

Comment:
In this case, you will create the same promise 2 times.
But they should be different.

![image](https://github.com/mate-academy/js_promise_basic_DOM/assets/84296041/9387327b-e113-498b-8c76-db02ba5e7294)